src/routes.tsx

```diff
@@ -15,6 +15,23 @@ import { LoginApp } from './client/pages/LoginApp'
 import { OfferNew } from './client/pages/OfferNew'
 import { SignLoading } from './client/pages/SignLoading'
 
+enum EmbarkStory {
+  DenmarkContents = 'Web Onboarding DK - Contents',
+  DenmarkContentsAccident = 'Web Onboarding DK - Danish Contents-Accident',
+
+  NorwayContentsNorwegian = 'Web Onboarding NO - Norwegian Contents',
+  NorwayContentsEnglish = 'Web Onboarding NO - English Contents',
+  NorwayTravelNorwegian = 'Web Onboarding NO - Norwegian Travel',
+  NorwayTravelEnglish = 'Web Onboarding NO - English Travel',
+  NorwayComboNorwegian = 'Web Onboarding NO - Norwegian Combo',
+  NorwayComboEnglish = 'Web Onboarding NO - English Combo',
+
+  SwedenNeederSwedish = 'Web Onboarding - Swedish Needer',
+  SwedenNeederEnglish = 'Web Onboarding - English Needer',
+  SwedenSwitcherSwedish = 'Web Onboarding - Swedish Switcher',
+  SwedenSwitcherEnglish = 'Web Onboarding - English Switcher',
+}
```

Comment:
Really nice to have these collected like this 👆 Just a question out of curiosity - what is the value here of using an enum instead of a plain ol' JS object?

Reply:
Hmm, no we are not using it to type anything so a JS object would work just as well in this case I believe